src/3rd_party_bundle_list.c

```diff
@@ -107,9 +107,41 @@ static bool parse_options(int argc, char **argv)
 	return true;
 }
 
+static void print_repo_header(const char *repo_name)
+{
+	const int HEADER = 18;
```

Comment:
where's this 18 came from? Can't it be replaced by a sizeof()?
magic numbers are usually hard to maintain.

Reply:
It is the length of " 3rd-Party Repo:  ", I'll modify this so we don't have magic numbers